### PR TITLE
GitHub sign-in support, front-end edition

### PR DESCRIPTION
**Pull request by jpeddicord**
Substantially simpler than clearlydefined/service#56.

### CODE REVIEW

File: src/api/clearlyDefined.js [hunk 1]
```diff
@@ -133,7 +124,7 @@ function post(url, token, payload) {
   return fetch(url, {
     headers: getHeaders(token),
     method: 'POST',
-    body: JSON.stringify(payload)
+    body: JSON.stringify(payload),
```
Comment (jeffmcaffer) [inline src/api/clearlyDefined.js]: is this a linting thing to _add_ trailing ','?

File: src/components/Header.js [hunk 2]
```diff
@@ -10,13 +10,12 @@ import { ROUTE_ROOT } from '../utils/routingConstants'
 import { Nav, Navbar, NavItem } from 'react-bootstrap'
 import { LinkContainer, IndexLinkContainer } from 'react-router-bootstrap'
 import { filter } from 'lodash'
-import { LoginPrompt } from './'
+import { url } from '../api/clearlyDefined';
```
Comment (jeffmcaffer) [inline src/components/Header.js]: ';'!  ;-)

File: src/api/clearlyDefined.js (hunk 1 above)
Comment (jpeddicord) [reply src/api/clearlyDefined.js]: Habit. Did we decide on a formatter for this repo? I don't see one. `prettier`?

Comment (mynameistechno) [reply src/api/clearlyDefined.js]: +1 for `prettier`!

Comment (jpeddicord) [reply src/api/clearlyDefined.js]: Agreed. I'll quick-fix these format issues you've noted in this review, but let's save the format pass for another commit.

**Pull request merged**